test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -69,6 +69,17 @@ std::string clustersJson(const std::vector<std::string>& clusters) {
   return fmt::sprintf("\"clusters\": [%s]", absl::StrJoin(clusters, ","));
 }
 
+void verifyCaresDnsConfigAndUnpack(
+    envoy::config::core::v3::TypedExtensionConfig& typed_dns_resolver_config,
```

Comment:
`typed_dns_resolver_config` can be const

Reply:
ACK